.jules/bolt.md

```
@@ -0,0 +1,3 @@
+## 2025-02-12 - Vectorizing audio frame iteration in streaming ASR
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)